bin/mbr_inspect.ml

```diff
@@ -29,33 +33,32 @@ let print_mbr_fields mbr =
       Printf.printf "    size_sectors: %ld\n" part.Mbr.Partition.sectors)
     mbr.partitions
 
-let read_mbr mbr =
-  let ic = open_in_bin mbr in
-  let buf = Bytes.create Mbr.sizeof in
-  let () = really_input ic buf 0 Mbr.sizeof in
-  close_in ic;
-  match Mbr.unmarshal (Cstruct.of_bytes buf) with
-  | Ok mbr -> mbr
-  | Error msg ->
-      Printf.printf "Failed to read MBR from %s: %s\n" mbr msg;
-      exit 1
+let read_mbr print_bootstrap_code mbrs =
+  List.iter
+    (fun mbr ->
+      let ic = open_in_bin mbr in
+      let buf = Bytes.create Mbr.sizeof in
+      let () = really_input ic buf 0 Mbr.sizeof in
+      close_in ic;
+      match Mbr.unmarshal (Cstruct.of_bytes buf) with
+      | Ok mbr -> print_mbr_fields print_bootstrap_code mbr
+      | Error msg ->
+          Printf.printf "Failed to read MBR from %s: %s\n" mbr msg;
+          exit 1)
+    mbrs
+
+let mbrs = Arg.(non_empty & pos_all file [] & info [] ~docv:"[DISK IMAGES]")
```

Comment:
`"[DISK IMAGES]"` collides a bit with two conventions of cmdliner: One is that things in square brackets are optional, and the other is that cmdliner marks arguments that can be passed more than once with `…` and it should be written in singular form. Space also doesn't work too well either.